mne_bids/write.py

```diff
@@ -338,7 +338,28 @@ def _events_json(fname, overwrite=False):
         Whether to overwrite the output file if it exists.
     """
     new_data = {
-        "sample": {"Description": "The event onset time in number of sampling points."},
+        "onset": {
+            "Requirement Level": "REQUIRED",
+            "Data type": "number",
+            "Description": (
+                "Onset (in seconds) of the event from the beginning of the first data point. "
+                "Negative onsets account for events before the first stored data point. "
+                "This column must appear first in the file."
```

Comment:
I don't think we need this here, as we're handling the column order ourselves

Reply:
Handled in https://github.com/mne-tools/mne-bids/pull/1255/commits/8895caff522d055ae25d2fcf55b833eccb2eda85